sdk/dynamicconfig/metric/controller/notify/mock/notifier.go

```diff
@@ -0,0 +1,43 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mock
+
+import (
+	"go.opentelemetry.io/contrib/sdk/dynamicconfig/metric/controller/notify"
+)
+
+type Notifier struct {
```

Comment:
Is this still a notifier? Or rather is a monitor for remote config changes?

Reply:
Yup, it does the same thing as Churan's original notifier, though replaces the observer-listener pattern with Go channels. Maybe it makes more sense to rename it to something like `ConfigMonitor` then?